### PR TITLE
Fixed right sidebar header

### LINKED ISSUE #605: Right Sidebar - the header should be fixed
The right sidebar header currently scrolls out of view. It should be fixed on the top so that the commands are always available.

We can probably do more to improve the scrolling behavior of the accordion, like making the accordion item headers sticky, so the top most item header is always visible, but that is probably out of scope for now. 

screen shot 2016-08-28 at 2 01 23 pm

screen shot 2016-08-28 at 2 01 11 pm

### PR DESCRIPTION
**Pull request by wldcordeiro**
Fixes #605

Simple change of the container to make it a flex container and add the proper scroll support for the accordion.

### CODE REVIEW

File: public/js/components/RightSidebar.css [hunk 1]
```diff
@@ -1,13 +1,19 @@
 .right-sidebar {
+  display: flex;
+  flex-direction: column;
   flex: 1;
   white-space: nowrap;
 }
 
+.right-sidebar .accordion {
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```
Comment (wldcordeiro) [inline public/js/components/RightSidebar.css]: May not need this `-webkit` property here, but might be useful in cases for outside Firefox?

Comment (jasonLaster) [inline public/js/components/RightSidebar.css]: yeah - i don't think we need that property :)

**Comment by jasonLaster** [comment]
one nit. otherwise ready to go

File: public/js/components/RightSidebar.css (hunk 1 above)
Comment (wldcordeiro) [inline public/js/components/RightSidebar.css]: 👍

**Comment by wldcordeiro** [comment]
Nit addressed and commit amended. 😄

**Comment by jasonLaster** [comment]
💃  in

**Pull request merged**